front/components/assistant/conversation/ConversationParticipants.tsx

```diff
@@ -0,0 +1,83 @@
+import { Avatar } from "@dust-tt/sparkle";
+import React from "react";
+
+import { ConversationType } from "@app/types/assistant/conversation";
+
+export function ConversationParticipants({
+  conversation,
+}: {
+  conversation: ConversationType;
+}) {
+  type UserParticipant = {
```

Comment:
Not sure if we want this to be forbidden but as they are only used here and have no point being exported I put them directly in the component!